client/src/components/Cell/Cell.jsx

```diff
@@ -24,6 +24,7 @@ const Cell = (props) => {
       {props.content === 'none' && <SvgNone />}
       {props.content === 'bomb' && <SvgBomb />}
       {props.content === 'kill' && <SvgDownShip />}
+      {props.content === 'miss' && <SvgEmpty />}
```

Comment:
It's better to use constants instead of strings, so that you don't make a typo.

Reply:
fixed